shotgun_api3/shotgun.py

```diff
@@ -2102,4 +2152,6 @@ def _translate_filters_simple(sg_filter):
 
     return condition
 
-     
+def _version_str(version):
+    """Converts a tuple of int's to a '.' separated str"""
+    return '.'.join(map(str, version))
```

Comment:
Can't believe that Python's `join` is called on the separator... It should be a functionality of an enumerable, not a string... Weird...!